xcat-inventory/xcclient/inventory/schema/1.0/osimage.yaml

```diff
@@ -14,14 +14,18 @@
       groups: "T{osimage.groups}"
     role: 
     - "T{osimage.profile}"
-    - "C:${{value=V{role}: True if str(value) in ('compute','service') else False}}"
+    - "C:${{value=V{role}: True if str(value) in ('','compute','service') else False}}"
```

Comment:
what's this for?

Reply:
some newly added osimage, such as cumulus, do not have attribute "role"

Reply:
ok, it is for cumulus osimage